src/main/java/bio/terra/service/filedata/azure/tables/TableServiceClientUtils.java

```diff
@@ -15,6 +17,9 @@
 
   public static boolean tableHasEntries(
```

Comment:
Would it make sense to move these static methods to a class that contains a `TableServiceClient`? If that object's lifetime is long enough (that is, we wouldn't end up making a new one every time a method is called), it would be a better way to organize the code.

Reply:
I've assumed that we cannot pass around the tableServiceClient due to expiration concerns. For each method in the daos, we're building the client passed on azure account info, so I don't think there is a class that has the client.

Reply:
Yes, I believe it's only valid for the current call stack. But at any point in the call stack, it's valid for all calls underneath it. E.g., what if the code that created a `TableServiceClient` returned a wrapper that supported the API calls we end up making on it? The object would contain all other state we have up to that point which also provides context. I'm not suggesting you do this now, it's something to think about as a future refactor.